### PR TITLE
Update links in docs

### PR DESCRIPTION
**Pull request by micwoj92**

### CODE REVIEW

File: docs/customization/custom_configs.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ These are `autoexec.cfg`, which runs at game launch, and the 9 class configs
 `medic.cfg`, `sniper.cfg`, and `spy.cfg`) which run on switching to a new class.
 All of these files go into a folder called `user` you create in `tf/cfg`.
 
-If you don't have these files, you can download the [mastercomfig template](https://github.com/mastercomfig/mastercomfig/releases/latest/download/template.cfg) and move the `user` folder to your `tf/cfg/` folder to customize the config.
+If you don't have these files, you can download the [mastercomfig template](https://github.com/mastercomfig/mastercomfig/releases/latest/download/autoexec_template.cfg) and move the `user` folder to your `tf/cfg/` folder to customize the config.
```
Comment (mastercoms) [inline docs/customization/custom_configs.md]: This should be the template zip.

File: docs/next_steps/troubleshoot.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ The command will only run on the first time you spawn in a match.
 
 ## TF2 crashes when entering a Competitive Mode match
 
-According to the [Official FAQ](https://www.teamfortress.com/meetyourmatch/faq/), Competitive Mode requires DirectX 9. Make sure your `dxlevel` is set to 90 or above (see [here](../customization/launch_options/#dxlevel-launch-options) for instructions).
+According to the [Official FAQ](https://www.teamfortress.com/meetyourmatch/faq/), Competitive Mode requires DirectX 9. Make sure your `dxlevel` is set to 90 or above (see [here](https://docs.mastercomfig.com/en/latest/customization/launch_options/#dxlevel-launch-options) for instructions).
```
Comment (mastercoms) [inline docs/next_steps/troubleshoot.md]: maybe `../../customization/launch_options#dxlevel-launch-options` will work

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: docs/customization/custom_configs.md (hunk 1 above)
Comment (micwoj92) [reply docs/customization/custom_configs.md]: done

File: docs/next_steps/troubleshoot.md (hunk 2 above)
Comment (micwoj92) [reply docs/next_steps/troubleshoot.md]: I did that https://github.com/mastercomfig/mastercomfig/pull/359/commits/b47b2351746677795597bd657837054b54047970 , but i dont have option to test this (I mean I could compile the whole docs but too lazy) so this seemed like safest option, but will do again

Comment (micwoj92) [reply docs/next_steps/troubleshoot.md]: done

**Review by mastercoms — APPROVED** [review]

**Pull request merged**